packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -141,46 +179,85 @@ class SharingCollection extends DocumentCollection {
 SharingCollection.normalizeDoctype = DocumentCollection.normalizeDoctypeJsonApi
 
 // Rules determine the behavior of the sharing when changes are made to the shared document
-// See https://github.com/cozy/cozy-stack/blob/master/docs/sharing-design.md#description-of-a-sharing
-const getSharingRules = (document, sharingType) => {
-  const { _id, _type } = document
+// See https://docs.cozy.io/en/cozy-stack/sharing-design/#description-of-a-sharing
+export const getSharingRules = (document, sharingType) => {
+  if (sharingType) {
+    console.warn(
+      `sharingType is deprecated and will be removed. We now set this default policy : ${
+        isFile(document)
+          ? getSharingRulesForFile(document)
+          : getSharingRulesForPhotosAlbum(document)
+      }} \n      
+      If this default policy doesn't follow your need, we have to set a custom policy 
```

Comment:
```suggestion
      If this default policy doesn't follow your need, please set custom rules
```

* IMHO the use of "your" then "we" makes it confusing
* I do no think that adding the "policy" concept is useful ? Is there a difference with the rules ? It seems that a "policy" is a subset of a "rule" ? `getSharingPolicyForReferencedFiles` -> `getRuleAttributesForReferencedFiles` ?